lib/member_page.rb

```diff
@@ -8,7 +8,10 @@ class MemberPage < Page
   end
 
   field :area do
-    noko.xpath("id('Breadcrumb')/li")[-2].text.tidy
+    noko.xpath("id('Breadcrumb')/li")[-2]
+        .text
+        .sub('Representatives', 'Representative')
```

Comment:
I'm not quite following this substitution. If we're trying to get the area here, don't we simply want "Alderney" (rather than "Alderney Representative")?